interfaces/apparmor/backend.go

```diff
@@ -587,7 +588,7 @@ func (b *Backend) SandboxFeatures() []string {
 	}
 
 	features := kernelFeatures()
```

Comment:
Why isn't this `features, _ := kernelFeatures()`?

Reply:
Because for the sake of landing a smaller patch, originally, there's a compatibility function that ignores the error. It was present in the first iteration of this patch and I kept it. I have also a small set of patches on top of the initial one that discard that function.

Reply:
I removed the old function now.